python/tvm/micro/contrib/zephyr.py

```diff
@@ -408,6 +408,7 @@ def _get_nrf_device_args(self):
     BOARD_USB_FIND_KW = {
         "nucleo_f746zg": {"idVendor": 0x0483, "idProduct": 0x374B},
         "stm32f746g_disco": {"idVendor": 0x0483, "idProduct": 0x374B},
+        "nucleo_l4r5zi": {"idVendor": 0x0483, "idProduct": 0x374B},
```

Comment:
```suggestion
        "nucleo_l4r5zi": {"idVendor": 0x0483, "idProduct": 0x374B},
        "nucleo_f746zg": {"idVendor": 0x0483, "idProduct": 0x374B},
        "stm32f746g_disco": {"idVendor": 0x0483, "idProduct": 0x374B},
```

This is a massive nit, but I suggest we keep the boards, for our own sanity, alphabetically here, where possible.

Reply:
i agree with alphabetic sorting